wandb/sdk/wandb_verify.py

```diff
@@ -0,0 +1,493 @@
+"""
```

Comment:
Can we move this file to `wandb/sdk/verify/verify.py`

sdk/wandb_*.py was supposed to signify a public part of the sdk like wandb.init()

Since all this is an internal thing to support the commandline, and it doesnt really follow the normal rules, importing internal stuff gql etc, we can whitelist it as we enforce rules more clearly or refactor some interfaces to better support this type of use case.

Reply:
Absolutely. I appreciate the explanation